### ISSUE TITLE
Swiper.js Speed Decreases with Multiple Hover Actions

### ISSUE THREAD

**Issue by Sahitya1707** [nolimits4web/swiper#7685]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/sandbox/dry3lt?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clzwxi3wp0007356g2xnan6ho%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clzwxi3wo0003356g5h2e8f4a%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clzwxi3wp0004356goij3y8gw%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clzwxi3wp0006356gas9uisla%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clzwxi3wo0003356g5h2e8f4a%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clzwxi3wo0002356g5zsnowk7%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Findex.html%2522%257D%255D%252C%2522id%2522%253A%2522clzwxi3wo0003356g5h2e8f4a%2522%252C%2522activeTabId%2522%253A%2522clzwxi3wo0002356g5zsnowk7%2522%257D%252C%2522clzwxi3wp0006356gas9uisla%2522%253A%257B%2522id%2522%253A%2522clzwxi3wp0006356gas9uisla%2522%252C%2522activeTabId%2522%253A%2522clzwxiijv0012356g5phqxirw%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clzwxi3wp0005356gjxlmgzbg%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%257D%252C%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522id%2522%253A%2522clzwxiijv0012356g5phqxirw%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%252C%2522clzwxi3wp0004356goij3y8gw%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clzwxi3wp0004356goij3y8gw%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

### Bug description

I have created a swiper using swiper js and I have tried the method of using mouseenter stop and start (i don't know the exact naming but I haved used this and also the auto play start and stop) but it causing to stop only after one items gets outside of the current screen but I want to stop the items as soon as hover. And when my mouse get outside of that items that should be run smoothly again. I manages to achieved the stopping of the swiper as soon as hover using the transform and transtion duration but when my cursor is out of the swiper the speed of the swiper decrease i mean I have used the swiper.passedParams.speed to get the actual speed and also used the transform property to make it as it was before. but it giving me unexpected behaviour when my cursor is ot the speed decreases and if i hover multiple time speed will gradually decrease more.

### Expected Behavior

I want to stop the swiper as soon as I hover and start it again immedietely when the mouse is outside that swiper.

### Actual Behavior

`<script>
      var swiper = new Swiper(".mySwiper", {
        slidesPerView: "3",
        preventClicksPropagation: false,
        preventClicks: true,
        threshold: 10, // Adjust this value as needed
        spaceBetween: 0,
        loop: true,
        autoplay: {
          delay: 0,
          disableOnInteraction: false,
          pauseOnMouseEnter: false,
        },
        speed: 2000,
        allowTouchMove: false,
        preventClicks: true,
      });

      const wrapper_element = document.querySelector(`.swiper-wrapper`);
      wrapper_element.addEventListener("mouseover", () => {
        const current_transform =
          window.getComputedStyle(wrapper_element).transform;
        wrapper_element.style.transitionDuration = "0ms";
        wrapper_element.style.transitionDelay = "0ms";
        wrapper_element.style.transform = current_transform;
      });

      wrapper_element.addEventListener("mouseout", () => {
        wrapper_element.style.transitionDuration = `${swiper.passedParams.speed}ms`;
        wrapper_element.style.transitionDelay = " ";
        wrapper_element.style.transform = `translate3d(${swiper.translate}px, 0px, 0px)`;
      });
    </script>`

### Swiper version

11.1.9

### Platform/Target and Browser Versions

Chrome

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [X] I'm willing to open a PR

**Comment by skyfxxker** [comment]
Is the problem solved, buddy?

**Comment by Sahitya1707** [comment]
No, It's not solved instead I created custom ticker by myself. I didnot used swiper js.
